### PR TITLE
Fixes the activity log label for project extend and geojson

### PR DESCRIPTION
**Pull request by sergiogcx**
Preview:
https://deploy-preview-214--atd-moped-main.netlify.app/

The problem:

The fix:

To test, run locally with `$ npm run start:staging`

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectActivityLog.js [hunk 1]
```diff
@@ -128,11 +129,19 @@ const ProjectActivityLog = () => {
     return data?.moped_activity_log?.length ?? 0;
   };
 
+  /**
+   * Returns True if the field should be a generic type (i.e., maps, objects)
+   * @param {string} field - The field name (column name)
+   * @return {boolean} - True if the field is contained in the ProjectActivityLogGenericDescriptions object
+   */
+  const isFieldGeneric = field =>
+    field in ProjectActivityLogGenericDescriptions;
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectView/ProjectActivityLog.js:138]: I need to remember to use the `in` keyword more. 👍

File: moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js [hunk 2]
```diff
@@ -1305,6 +1305,15 @@ export const ProjectActivityLogOperationMaps = {
   },
 };
 
+export const ProjectActivityLogGenericDescriptions = {
+  project_extent_ids: {
+    label: "Project extent updated",
+  },
+  project_extent_geojson: {
+    label: "Project geo-json updated",
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js:1313]: Maybe this should be GeoJSON instead of geo-json?

**Review by mddilley — APPROVED** [review]
Looks great, Sergio! I like this way of controlling what outputs in the activity log. A nice enhancement to an already amazing feature! 😎

File: moped-editor/src/views/projects/projectView/ProjectActivityLog.js (hunk 1 above)
Comment (sergiogcx) [reply moped-editor/src/views/projects/projectView/ProjectActivityLog.js:138]: I found out it works out of luck, that's the syntax in python, it totally works in JavaScript too! :)

File: moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js (hunk 2 above)
Comment (sergiogcx) [reply moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js:1313]: Good call, will change now

**Review by amenity — APPROVED** [review]
LSTM!! ✨

**Pull request merged**